File: TsogosunProfileAdminWeb/src/app/model/Dtos/ms-patron-details/unconfirmedGMSPatronDetailsInfoDto.ts
import { PatronDetailsInfoDto } from "./patronDetailsInfoDto";

export interface UnconfirmedGMSPatronDetailsInfoDto extends PatronDetailsInfoDto {
    pcId?: number | undefined;
    documentTypeId?: number | undefined;
    documentExpireDate?: Date | undefined;
    pippep?: number | undefined;
}

File: TsogosunProfileAdmin/MSPatronDetails/Model/Dtos/UnconfirmedGMSPatronDetailsInfoDto.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace tsogosun.com.MSPatronDetails.Model.Dtos
{
    public class UnconfirmedGMSPatronDetailsInfoDto : PatronsDetailsInfoDto
    {		
		[NotMapped]
		public int? DocumentTypeId { get; set; }
		[NotMapped]
		public DateTime? DocumentExpireDate { get; set; }
		[NotMapped]
		public int? Pippep { get; set; }
	}
}
